test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -131,14 +91,14 @@ public class ContentResultTest
                         new MediaTypeHeaderValue("text/foo"),
                         "κόσμε",
                         null,
-                        "text/foo; charset=utf-8",
+                        "text/foo",
```

Comment:
Because we do not want to modify what the user has set explicitly, which is just `text/foo` without any charset